plat/allwinner/common/sunxi_bl31_setup.c

```diff
@@ -34,6 +35,17 @@ void bl31_early_platform_setup2(u_register_t arg0, u_register_t arg1,
 	console_16550_register(SUNXI_UART0_BASE, SUNXI_UART0_CLK_IN_HZ,
 			       SUNXI_UART0_BAUDRATE, &console);
 
+#ifdef BL32_BASE
```

Comment:
I’d use the “C if” over the preprocessor. That way, the compiler will check all the code.

Reply:
If BL32_BASE is undefined, the compiler will complain at line 46. Hence this is the pattern we tend to use.

Reply:
I thought it’s unconditionally defined. But I overlooked `#ifndef SPD_none`.